libparsec/crates/client/src/client/mod.rs

```diff
@@ -151,8 +151,20 @@ impl Client {
                 start_workspaces_refresh_list_monitor(client.event_bus.clone(), client.clone())
                     .await;
 
-            let user_sync_monitor =
-                start_user_sync_monitor(client.user_ops.clone(), client.event_bus.clone()).await;
+            // TODO: rework user manifest sync once needed
```

Comment:
Is there a related issue ?

Reply:
No, because we just don't need user manifest sync for now

In theory we should just remove entirely the user sync stuff, however it's a lot of code that would be needed again as soon as we want to share configuration between a user's devices (while slightly modified given we no longer can use the realm for this use).

So in the end I think it's just the pragmatical approach to let this code "as is" for the moment (and hence why there is no issue, there is nothing to do for the moment, not even removal ^^)

Reply:
IMHO we could have an issue to 1/ link in all the places that are impacted 2/ centralize and document that discussion. But at the end of the day, it's project management boilerplate ^^